package.json

```diff
@@ -31,6 +31,7 @@
     "npm": ">=7.21.0"
   },
   "dependencies": {
+    "@discordjs/builders": "^0.6.0",
```

Comment:
I generally prefer to lock these version to prevent NPM craziness. There have been a few recorded instances of minor versions introducing breaking changes and even a security incident with a compromised package and new version.

Reply:
Removing the ^ & ~ for now for better package stability, though I wonder if this is an issue since we build the app via `npm ci`